src/roles.json

```
@@ -1553,21 +1553,40 @@
     "team": "demon",
     "firstNight": 13,
     "firstNightReminder": "The Lleech points to a player. Place the Poisoned reminder token.",
-    "otherNight": 32,
+    "otherNight": 33,
     "otherNightReminder": "The Lleech points to a player. That player dies.",
     "reminders": ["Dead",
         "Poisoned"],
     "setup": false,
     "ability": "Each night*, choose a player: they die. You start by choosing an alive player: they are poisoned - you die if & only if they die."
   },
+  {
+    "id": "alhadikhia",
+    "name": "Al-Hadikhia",
+    "edition": "",
+    "team": "demon",
+    "firstNight": 0,
+    "firstNightReminder": "",
+    "otherNight": 31,
+    "otherNightReminder": "The Al-Hadikhia chooses 3 players. Announce the first player, wake them to nod yes to live or shake head no to die, kill or resurrect accordingly, then put to sleep and announce the next player. If all 3 are alive after this, all 3 die.",
+    "reminders": [
+
+        1,
+        2,
+        3
+
+    ],
```

```suggestion
    "reminders": [1, 2, 3],
```
No need for that much padding. 😉 
Also, what about this?
> Online, this means you will need to not actually shroud the players if using clocktower.online, you will instead need to use the “Chose death” and “Chose life” reminder tokens in the tool as an alternative